docker-compose.yml

```
@@ -0,0 +1,30 @@
+version: '${TAG_COMPOSE}'
+services:
+  db:
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+      POSTGRES_HOST_AUTH_METHOD: trust
+    image: 'postgres:${TAG_POSTGRES}'
+    ports:
+      - '5432:5433'
+    restart: always
+    volumes:
+      - 'pgdata:/var/lib/postgresql/data'
+  web:
+    build: 
+      context: .
+      target: "dev"
+    depends_on:
+      - db
+    environment:
+      MIX_ENV: dev
+    env_file:
+      - .env
+    ports:
+      - '4000:4000'
+    volumes:
+      - .:/app
```

@keliumJU  I suggest these changes and organization:
```suggestion
    mintacoin_api:
        container_name: mintacoin_api
        build:
          context: .
          target: dev
        volumes:
          - .:/app
        ports:
          - 4000:4000
        environment:
          - CONTAINER_HOST=mintacoin_api
          - POSTGRES_USER=postgres
          - POSTGRES_PASSWORD=postgres
          - POSTGRES_DB_LOCAL=mintacoin_dev
          - POSTGRES_DB_TEST=mintacoin_test
          - POSTGRES_PORT=5432
          - POSTGRES_HOST=mintacoin_db
        depends_on:
          - mintacoin_db
        networks:
          - mintacoin
```

---

ok, change made and I like this